protocol/contracts/beanstalk/silo/SiloFacet/MigrationFacet.sol

```diff
@@ -55,5 +48,28 @@ contract MigrationFacet is ReentrancyGuard {
         LibLegacyTokenSilo._migrateNoDeposits(account);
     }
 
+    //////////////////////// CLAIM ////////////////////////
+
+    // as of the zero withdraw update, these functions are no longer used. 
+    // However, these are kept for backwards compatability.
+    function claimWithdrawal(
```

Comment:
I previously wrote natspec comments for both of these functions, take a look back to before they were removed and reintroduce them with the following changes:

- Replace the @notice line with `@notice DEPRECATED: ...`
- Add or extend the @dev comment explaining why these were deprecated.

When explaining deprecation, say something like:
> The Zero Withdraw update removed the two-step withdraw & claim process. These functions are left for backwards compatibility, to allow pending withdrawals from before the update to be claimed.